e2e_samples/parking_sensors/databricks/config/cluster.config.template.json

```diff
@@ -1,10 +1,11 @@
 {
     "cluster_name": "__REPLACE_CLUSTER_NAME__",
     "autoscale": { "min_workers": 1, "max_workers": 2 },
-    "spark_version": "6.6.x-scala2.11",
-    "autotermination_minutes": 120,
-    "node_type_id": "Standard_DS3_v2",
-    "driver_node_type_id": "Standard_DS3_v2",
+    "spark_version": "14.3.x-scala2.12",
+    "autotermination_minutes": 30,
+    "node_type_id": "Standard_DC4as_v5",
```

Comment:
Did we call this out in the documentation that the node types need to be available in the region and can be checked with `az vm list-usage --location "East US 2" -o table`